### PR TITLE
Initial commit of aodnfetcher

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,106 @@
 # python-aodnfetcher
 Multi-protocol artifact fetcher library/utility
+
+## Usage
```
Comment (akashisama) [inline README.md]: Should probably have an 'installation' heading as well

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+# Rope project settings
+.ropeproject
+
+# mkdocs documentation
+/site
+
+# mypy
+.mypy_cache/
+
+/.idea/vcs.xml
```
Comment (akashisama) [inline .gitignore]: Might be better to ignore the idea folder and anything in it

```
.idea
.idea/*

Comment (ghost) [reply .gitignore]: That is just the autogenerated gitignore, and for some reason they only ignore part of the .idea directory. I don't disagree with you though, is annoying to see IDE specific nonsense in the repo.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: No problem. It's entirely standard in terms of packaging/installing, but it never hurts to add a "quick start" type reminder.

**Comment by ghost** [comment]
@akashisama both changes made

**Pull request merged**